bin/init.sh

```diff
@@ -37,7 +37,7 @@ then
 fi
 
 declare -a files_with_port=(.env Dockerfile README.md src/main/resources/application.yaml)
-declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md web.config)
+declare -a files_with_slug=(build.gradle docker-compose.yml Dockerfile README.md web.config, src/main/uk/gov/hmcts/reform/demo/controllers/HelloWorldEndpoint.java)
```

Comment:
... and now you've lost me. Please, explain?

Reply:
please review #30 . if any questions feel free to DM

Reply:
Comma?

Reply:
😮. my bad